Makefile

```diff
@@ -46,3 +54,27 @@ test-bindings: $(PKG_BINDINGS_DIR)
 
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
+
+# Generate an image for containerized builds
+# Note: This image is local only
+image: Dockerfile
```

Comment:
Can we call this target `.image` instead of just `image` so that it is hidden from the tab completion?

Reply:
I can do that.